### PR TITLE
feat: initial setup of remixx

### PR DESCRIPTION
**Pull request by ScriptedAlchemy**
exporting easy-redux out, pulling in their examples for the time being - likely we will want to modify and be able to ensure Remixx works with all existing examples anyways.

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
Will start running code tomorrow, did a lot of cleaning so chances are there's a bug or two - once i start writing the babel plugin ill see if i need some patches.

File: src/babel/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+function babelPlugin() {
```
Comment (faceyspacey) [inline src/babel/index.js]: yay!

File: src/babel/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  return {
+    visitor: {
+      // eslint-disable-next-line
+      Program: function Program(path, { opts }) {
+        // remixx magic
```
Comment (faceyspacey) [inline src/babel/index.js]: more yay!

**Comment by faceyspacey** [comment]
excellent. that makes a lot of sense. we'll rewrite each example according to the Respond component API.

**Pull request merged**